File: plugin/spigot/src/main/java/com/jonathanpotts/cartographytable/spigot/CartographyTable.java
package com.jonathanpotts.cartographytable.spigot;

import org.bukkit.plugin.java.JavaPlugin;

/**
 * The CartographyTable server plugin.
 */
public class CartographyTable extends JavaPlugin {
  @Override
  public void onEnable() {
    getCommand("refresh-map-data").setExecutor(new CommandRefreshMapData(this));
  }
}

File: plugin/spigot/src/main/kotlin/com/jonathanpotts/cartographytable/spigot/CartographyTable.kt
package com.jonathanpotts.cartographytable.spigot

import org.bukkit.plugin.java.JavaPlugin

/**
 * The CartographyTable server plugin.
 */
@Suppress("unused")
class CartographyTable : JavaPlugin() {
    override fun onEnable() {
        getCommand("refresh-map-data")?.setExecutor(CommandRefreshMapData(this))
    }
}
